### PR TITLE
Handle windows conda library directories in default blas

### LINKED ISSUE #515: BLAS warning on Windows builds on conda-forge
### Description

```python
WARNING (pytensor.tensor.blas): Using NumPy C-API based implementation for BLAS functions.
```

See [the logs](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=829292&view=logs&j=171a126d-c574-5c8c-1269-ff3b989e923d&t=623c2ba8-c4f1-582d-6c78-1c7699e2e0a2&l=1897).

I'm going to add some more debug commands to try and get more info.

### PR DESCRIPTION
**Pull request by lucianopaz**
Closes #515

## Major / Breaking Changes
- ...

## New features
- ...

## Bugfixes
- Fix conda C library detection on Windows

## Documentation
- ...

## Maintenance
- ...

### CODE REVIEW

File: pytensor/link/c/cmodule.py [hunk 1]
```diff
@@ -2793,6 +2795,14 @@ def check_libs(
 
     cxx_library_dirs = get_cxx_library_dirs()
     searched_library_dirs = cxx_library_dirs + _std_lib_dirs
+    print(sys.platform, sys.prefix)
```
Comment (michaelosthege) [inline pytensor/link/c/cmodule.py:2798]: leftover debugging print?

**Review by michaelosthege — COMMENTED** [review]
is that failing test a bug uncovered by a change in the graph?

**Review by twiecki — APPROVED** [review]

**Comment by lucianopaz** [comment]
> is that failing test a bug uncovered by a change in the graph?

The failing test happened because there were empty `blas__ldflags`

**Comment by ricardoV94** [comment]
Isn't it good that the test actually failed?

**Comment by ricardoV94** [comment]
Okay you changed the dprint one, that's fine

**Comment by lucianopaz** [comment]
The other failing test was caused by having one extra line in the print statement: the warning you get at import when no blas headers are available. That doesn’t seem to be a valid failure to me

**Comment by lucianopaz** [comment]
Just to be clear, this doesn’t close #508. That one looks more like a nixOS problem with libraries and I need more info to address it

**Comment by ricardoV94** [comment]
Yeah, I just wonder if we will be blind about regressions, not that this test should have that responsibility

**Comment by ricardoV94** [comment]
@michaelosthege can you locally try on your Windows machine?

`>>python -c "import pytensor; print(pytensor.config.blas__ldflags)"`

**Comment by michaelosthege** [comment]
> @michaelosthege can you locally try on your Windows machine?
> 
> `>>python -c "import pytensor; print(pytensor.config.blas__ldflags)"`

```
-L"C:\Users\osthege\AppData\Local\mambaforge\envs\ptenv\Library\bin" -L"C:\Users\osthege\AppData\Local\mambaforge\envs\ptenv" -llapack -lblas -lcblas -lm -Wl,-rpath,C:\Users\osthege\AppData\Local\mambaforge\envs\ptenv\libs
```

**Pull request merged**